.github/workflows/rws-submissions-checks.yml

```
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 name: RWS submission checks
-on: pull_request_target
+on: 
+  schedule:
+  - cron: '0 14 * * *' #triggers job at 9am EST each day
```

We don't look at the PRs except on Tuesdays, so I don't think we need to have daily jobs. Just once weekly, before we review the PRs would be sufficient.

(I also think it'd be good to be precise about the time/timezone, so that it's clearer what the crontab is supposed to be.)

```suggestion
  - cron: '0 14 * * 2' # 2pm UTC each Tuesday
```